telegram/request/_httpxrequest.py

```diff
@@ -142,6 +149,7 @@ def __init__(
             )
 
         self._http_version = http_version
+        self._media_write_timeout = media_write_timeout
```

Comment:
don't we need an entry for this in the `__slots__`? 

The tests once again didn't pick it up cause of the same reason in #4130 ..